src/mixer.erl

```diff
@@ -99,11 +102,26 @@ group_mixins({_CMod, _}, [#mixin{mod=Mod, line=Line}=H|T], Accum) ->
 expand_mixin(Line, Name) when is_atom(Name) ->
     case catch Name:module_info(exports) of
         {'EXIT', _} ->
-            io:format("~s:~p Unable to resolve imported module ~p~n", [get_file_name(), Line, Name]),
+            io:format(
+                "~s:~p Unable to resolve imported module ~p~n",
+                [get_file_name(), Line, Name]),
+            exit({error, undef_mixin_module});
+        Exports ->
+            [#mixin{line=Line, mod=Name, fname=Fun, alias=Fun, arity=Arity}
+             || {Fun, Arity} <- Exports, Fun /= module_info]
+    end;
+expand_mixin(Line, {Name, except, Funs}) when is_atom(Name),
+                                              is_list(Funs) ->
+    case catch Name:module_info(exports) of
```

Comment:
Remove the dynamic function call on line 115. Only modules that define callbacks should make dynamic calls.